### DISCUSSION TITLE
Announcement for specific users or languages

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#10939] [Q&A] [answered]
Lately I learned about the announcement feature.
Is there also a way to reach specific translators or translators of a language via such a feature?

The point is I don't have contact information of all my subscribers. Most of them I just know there account name and they have an empty profile.

**Comment by nijel** [comment]
Yes, you can post announcement on several scopes - project, language, language in a project, component and translation (I'm not sure with the last two, I might be wrong here).

**Reply by buhtz** [reply]
Nice to know. How? It is not obvious to me looking into the UI.

**Reply by nijel** [reply, marked as answer]
It depends on where you post it. If you post it on project level, it is project scoped, if on translation, it is translation scoped. I can see this is hard to find...

**Reply by buhtz** [reply]
There is no "Announcment" option on Project-Language level but on Project-Component-Language level. Mhm... OK. That is a start. I would suggest that the Announcment dialog should indicate clearly its current scope.

**Reply by emmapeel2** [reply]
Yeah, an announcement in Project/Language would be nice, I also missed this possibility when doing announcements.

**Reply by nijel** [reply]
Indeed this is missing, I've created https://github.com/WeblateOrg/weblate/issues/11003 to track this feature.

**Reply by emmapeel2** [reply]
Another place I have wanted to post announcements is Category.

**Reply by nijel** [reply]
@emmapeel2 Makes sense, please open a separate issue as that one would also need a model change.

**Reply by emmapeel2** [reply]
I have opened https://github.com/WeblateOrg/weblate/issues/11106 for the announcements in Category
